packages/react/src/create-strict-api/__tests__/__fixtures__/strict-api.ts

```diff
@@ -10,6 +10,15 @@ interface CSSPropertiesSchema {
   bkgrnd: 'red' | 'green';
 }
 
-const { css, XCSSProp, cssMap, cx } = createStrictAPI<CSSPropertiesSchema>();
+type MediaQuery =
```

Comment:
We don't really test this limits other queries, right?

Reply:
Good to add, thanks